.circleci/config.yml

```diff
@@ -649,13 +649,6 @@ jobs:
             # 2. Branch name if 'hotfix' in branch name (branch name passed in via $CIRCLE_BRANCH)
             # 3. 'latest', which is the default
             echo "export IMAGE_TAG=`[ $CIRCLE_TAG ] && echo $(echo $CIRCLE_TAG | cut -d@ -f3) || [[ "$CIRCLE_BRANCH" =~ (hotfix) ]] && echo $CIRCLE_BRANCH || echo "latest" `" | tee -a $BASH_ENV
```

Comment:
intentional?

Reply:
yeah this wasn't working unf - @csjiang will add the fix back